serveur_calcul_python/classes/parking_reg_sets.py

```
@@ -124,36 +263,88 @@ def expand_land_use_table(self):
+        '''
+        # get_parking_reg_by_id
+        Returns a ParkingRegulations object whose id matches the one specified 
+        in the inputs
+        
+        Inputs:
+            - reg_id: Id of the ParkingRegulation you want to obtain
+        Outputs:
+            - ParkingRegulation containing the relevant ParkingRegulation
+        '''
+        out = super().get_reg_by_id(reg_id)
+        return out
```

Ruff (0.15.20)

[warning] Unnecessary assignment to `out` before `return` statement

Remove unnecessary assignment

(RET504)